Build fails: conversation.go undefined: post

Today, I tried to build with docker and it replied as follow:

github.com/nlopes/slack/conversation.go:137:8: undefined: post


It was ok yesterday. 
And I notice there is a commit few hours ago.

--- a/conversation.go
+++ b/conversation.go
@@ -134,7 +134,7 @@ func (api *Client) GetConversationsForUserContext(ctx context.Context, params *G
 		ResponseMetaData responseMetaData `json:"response_metadata"`
 		SlackResponse
 	}{}
-	err = post(ctx, api.httpclient, "users.conversations", values, &response, api.debug)
+	err = postSlackMethod(ctx, api.httpclient, "users.conversations", values, &response, api.debug)
 	if err != nil {
 		return nil, "", err
 	}

